notebooks/05a Modelling - regression.ipynb

```diff
@@ -389,15 +371,19 @@
    "source": [
     "## Train/test split\n",
     "\n",
-    "For model evaluation, we will hold back a 25% test set, and use cross-validation on the remaining 75% for all models until the final comparison is made."
+    "For model evaluation, we will hold back a 25% test set, and use cross-validation on the remaining 75% for all models until the final comparison is made.\n",
+    "\n",
+    "This helps improve the robustness of the models by reducing the risk of overfitting across the entire dataset.\n",
```

Comment:
I would characterise the purpose of the test set as more being about enabling you to assess the performance of the model on a data set it has not be been exposed to before. Cross-validation checks for overfitting (as you have noted below)